xclim/indices.py

```diff
@@ -991,8 +991,8 @@ def max_n_day_precipitation_amount(pr, window=1, freq='YS'):
     """
 
     # rolling sum of the values
-    arr = pr.rolling(time=window, center=False).sum(dim='time')
-    return arr.resample(time=freq).max(dim='time')
+    arr = pr.rolling(time=window, center=False).sum()
+    return arr.resample(time=freq).max()
```

Comment:
Just to be clear, the latest version of xarray sets the default aggregation dimension for the rolling and resample operations?

Reply:
not resample. Sorry my mistake will adjust and resubmit.